### PR TITLE
Update definitions-terms.md

### PR DESCRIPTION
**Pull request by Zhurik**
typo

### CODE REVIEW

**Comment by imjasonh** [comment]
Both of these spellings seem to be accepted as correct in common usage. I'm not sure this change is worth making.

**Review by sudo-bmitch — CHANGES_REQUESTED** [review]
Implementer appears to be the correct term: https://dictionary.cambridge.org/us/dictionary/english/implementer

Changing this would also require renaming the file and the link to the file.

File: definitions-terms.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ OCI Image is a specific type of artifact. However, an OCI image is not meant to
 
 ## Registry
 
-A registry, or container registry, is an instance of the [distribution-spec]. See [Implementers][implementers] for a list of registries that support OCI Artifacts.
+A registry, or container registry, is an instance of the [distribution-spec]. See [Implementors][implementors] for a list of registries that support OCI Artifacts.
```
Comment (imjasonh) [inline definitions-terms.md:39]:
```suggestion
A registry, or container registry, is an instance of the [distribution-spec]. See [Implementers][implementers] for a list of registries that support OCI Artifacts.
```

This reverts most of this change, except the part that makes the link line up with the link listed at the bottom.

**Review by mikebrow — COMMENTED** [review]
pls rename the file :)

**Review by SteveLasker — APPROVED** [review]
Thank you, @Zhurik 
Appreciate the changes
LGTM
Steve

File: definitions-terms.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ OCI Image is a specific type of artifact. However, an OCI image is not meant to
 
 ## Registry
 
-A registry, or container registry, is an instance of the [distribution-spec]. See [Implementers][implementers] for a list of registries that support OCI Artifacts.
+A registry, or container registry, is an instance of the [distribution-spec]. See [Implementers](implementers.md) for a list of registries that support OCI Artifacts.
```
Comment (mikebrow) [inline definitions-terms.md:39]: believe it or not the typical pattern for linking to the bottom of the md is [][] 

though linking directly the file with () also works might as well use the redirection to below

```suggestion
A registry, or container registry, is an instance of the [distribution-spec]. See [Implementers][implementers] for a list of registries that support OCI Artifacts.
```

Comment (SteveLasker) [reply definitions-terms.md:39]: @Zhurik, we appreciate the effort for the PR. If the additional tweaks are too much detail, we can merge this as is, and tweak a bit further as the link works with your PR. We just have a dangling entry in the [links section at the bottom](https://github.com/opencontainers/artifacts/pull/64/files#diff-da55140985df1701b1b0ece67f61bfde0bc2e2e85aeea1cb3d9bba978945b1deL51) of the file.

**Review by mikebrow — APPROVED** [review]
/LGTM we can fix the reference to link at bot of file in another commit.. cheers

**Comment by SteveLasker** [comment]
Hi @Zhurik,
We can do the link fix post merge, however, unfortunately, we need a signature associated with your PR
See: https://github.com/opencontainers/artifacts/blob/main/CONTRIBUTING.md#sign-your-work for OCI Contributing guidelines.

And the DCO fix:
See https://github.com/opencontainers/artifacts/pull/64/checks?check_run_id=13574434672
```
git config --global user.name "full name here"
git config --global user.email "email address here" 
git commit -s --amend
```

**Pull request merged**